src/stories/components/ShowCarousel.stories.ts

```diff
@@ -8,7 +8,7 @@ const meta = {
     component: ShowCarousel,
     tags: ['autodocs'],
     parameters: {
-        layout: 'centered',
+        layout: 'fullscreen',
```

Comment:
What was the reason for updating this?

Reply:
Changed for testing, forgot to change it back.